gcpkms/keysource.go

```diff
@@ -220,12 +222,18 @@ func (key *MasterKey) newKMSClient() (*kms.KeyManagementClient, error) {
 		opts = append(opts, option.WithCredentialsJSON(key.credentialJSON))
 	default:
 		credentials, err := getGoogleCredentials()
-		if err != nil {
-			return nil, err
-		}
 		if credentials != nil {
 			opts = append(opts, option.WithCredentialsJSON(credentials))
 		}
+
+		at_credentials, at_err := getGoogleAccessToken()
+		if at_credentials != nil {
```

Comment:
Related to my other comment, given precedence to the default credentials is it even worth running this if we already found credentials?